content/covidcast/export.md

```diff
@@ -5,7 +5,6 @@ description: Use COVIDcast data in your own analysis
 layout: covidcast_app
 app_mode: export
 order: 6
-icon: solid/download
```

Comment:
what does this change do?

Reply:
it shouldn't have any effect, the line is a leftover an old version where the icons were still used